test-network-nano-bash/peer1.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+#look for binaries in local dev environment /build/bin directory and then in local samples /bin directory
+export PATH="${PWD}"/../../fabric/build/bin:"${PWD}"/../bin:"$PATH"
+export FABRIC_CFG_PATH="${PWD}"/../config
+
+export FABRIC_LOGGING_SPEC=debug:cauthdsl,policies,msp,grpc,peer.gossip.mcs,gossip,leveldbhelper=info
+export CORE_PEER_TLS_ENABLED=true
+export CORE_PEER_TLS_CERT_FILE="${PWD}"/crypto-config/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/server.crt
+export CORE_PEER_TLS_KEY_FILE="${PWD}"/crypto-config/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/server.key
+export CORE_PEER_TLS_ROOTCERT_FILE="${PWD}"/crypto-config/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt
+export CORE_PEER_ID=peer0.org1.example.com
+export CORE_PEER_ADDRESS=127.0.0.1:7051
+export CORE_PEER_LISTENADDRESS=127.0.0.1:7051
+export CORE_PEER_CHAINCODEADDRESS=host.docker.internal:7052
```

Comment:
I needed to change `host.docker.internal` to `127.0.0.1`

Reply:
127.0.0.1 didn't work for me. It took me a while to figure this one out, but the docker container couldn't get to the host with 127.0.0.1. I then found out that host.docker.internal resolves to the local host - see https://docs.docker.com/docker-for-mac/networking/.  But that probably works on Mac only. What OS do you have?

Reply:
... ah.. yes this is a variation in Docker between Mac and Linux.